manuscript/chapter1.md

```diff
@@ -232,116 +217,120 @@ npm test
 
 ### Exercises:
 
-* `npm start` your application and visit the application in your browser (you can exit the command by pressing Control + C)
-* run the interactive `npm test` script
-* run the `npm run build` script and verify that a *build/* folder was added to your project (you can remove it again afterward; note that the build folder can be used later on to [deploy your application](https://www.robinwieruch.de/deploy-applications-digital-ocean/))
-* make yourself familiar with the folder structure
-* make yourself familiar with the content of the files
-* read more about [the npm scripts and create-react-app](https://github.com/facebookincubator/create-react-app)
+* `npm start` your application and visit the application in your browser (Exit the command by pressing Control + C)
+* Run the `npm test` script
+* Run the `npm run build` script and verify that a *build/* folder was added to your project (you can remove it afterward. Note that the build folder can be used later on to [deploy your application](https://www.robinwieruch.de/deploy-applications-digital-ocean/))
+* Familiarize yourself with the folder structure
+* Check the content of the files
+
+**More Reading**
+* About [npm scripts and create-react-app](https://github.com/facebookincubator/create-react-app)
 
 ## Introduction to JSX
 
-Now you will get to know JSX, which is the syntax in React. As mentioned before, *create-react-app* has already bootstrapped a boilerplate application for you. All files come with default implementations. In the beginning, the only file you will touch is the *src/App.js* file.
+Now we will get to know JSX, the syntax in React. As mentioned before, *create-react-app* has already bootstrapped a basic application for you, and all files come with their own default implementations. For now, the only file we will modify is the *src/App.js* file.
 
-{title="src/App.js",lang=javascript}
+{title="src/App.js",lang="javascript"}
 ~~~~~~~~
 import React, { Component } from 'react';
 import logo from './logo.svg';
 import './App.css';
 
 class App extends Component {
-  render() {
-    return (
-      <div className="App">
-        <header className="App-header">
-          <img src={logo} className="App-logo" alt="logo" />
-          <h1 className="App-title">Welcome to React</h1>
-        </header>
-        <p className="App-intro">
-          To get started, edit <code>src/App.js</code> and save to reload.
-        </p>
-      </div>
-    );
-  }
+ render() {
+  return (
+   <div className="App">
+    <header className="App-header">
+     <img src={logo} className="App-logo" alt="logo" />
+     <h1 className="App-title">Welcome to React</h1>
+    </header>
+    <p className="App-intro">
+     To get started, edit <code>src/App.js</code> and save to reload.
+    </p>
+   </div>
+  );
+ }
```

Comment:
Note sure about the indentation here, I think it should be 2 spaces.

Reply:
Which line exactly?